test/TokenCuratedRegistry.behavior.js

```diff
@@ -129,6 +129,14 @@ function shouldBehaveLikeTokenCuratedRegistry ({
         })
       })
 
+      describe('when challenge.fundsRequired() is greater than the challenge stake', function () {
+        it('reverts', async function () {
+          await this.registry.add(itemData)
+          await this.challengeFactory.mock_set_fundsRequired(minStake * 1.5)
+          await this.token.approve(this.registry.address, minStake, { from: challenger })
```

Comment:
@emaG3m this should approve `minStake * 1.5`, otherwise this test would still pass if the require check for the fundsRequired max was removed.

Reply:
I was able to completely isolate this test to the new logic and the fact that it's greater than minDeposit. The challenger should still only be approving the minDeposit. The registry has more tokens from the itemOwner's deposit

Reply:
oops, you're right! I just read it wrong